Source/Core/Core/CoreTiming.cpp

```diff
@@ -50,7 +50,7 @@ static Common::FifoQueue<BaseEvent, false> tsQueue;
 // event pools
 static Event *eventPool = nullptr;
 
-static float lastOCFactor;
+float lastOCFactor;
```

Comment:
It would be really nice if this variable wasn't a global, along with the other globals in this file.